web/modules/mof/src/Form/ModelSearchForm.php

```diff
@@ -30,9 +30,11 @@ public function getFormId() {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state) {
+  public function buildForm(array $form, FormStateInterface $form_state, string $collection_name = NULL) {
     $req = $this->request;
 
+    $form['collection'] = $collection_name;
```

Comment:
@gregcube This works but may not be the best way to pass this around. Any suggestion to do that better welcome! Thanks.

Reply:
I'll take a look and follow up soon.

Reply:
We don't need to pass $collection_name at all.
We can pull the route name from the symfony request object.